pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotSegmentRestletResource.java

```diff
@@ -1130,4 +1152,38 @@ private SuccessResponse updateZKTimeIntervalInternal(String tableNameWithType) {
       }
       return new SuccessResponse("Successfully updated time interval zk metadata for table: " + tableNameWithType);
   }
+
+  public List<Pair<TableType, List<String>>> selectSegments(
```

Comment:
Possible to refactor `getSelectedSegments()` with the same util function as well?